lib/phoenix/live_dashboard/system_info.ex

```diff
@@ -218,10 +228,23 @@ defmodule Phoenix.LiveDashboard.SystemInfo do
 
     next_state = for {_sorter, info} <- processes, into: %{}, do: {info[:pid], info[:reductions]}
 
-    count = if search, do: length(processes), else: :erlang.system_info(:process_count)
+    count = length(processes)
     processes = processes |> Enum.sort() |> Enum.take(limit) |> Enum.map(&elem(&1, 1))
 
-    {processes, count, next_state}
+    {active_filter, available_filters, processes, count, next_state}
+  end
+
+  def get_process_filter_data(filter) do
+    case Application.get_env(:phoenix_live_dashboard, :process_filter) do
```

Comment:
In this case, it is better to use https://hexdocs.pm/elixir/Application.html#fetch_env/2; so we can differentiate between a `nil` value or not found situation.

Reply:
I think that both `nil` and `not found` mean the same, that is, no process filter module. Can't see a case where we would want to differentiate between them.